apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -51,6 +52,35 @@ For detailed help about a specific command, use: rush <command> -h
 "
 `;
 
+exports[`CommandLineHelp prints the help for "rush add" 1`] = `
+"usage: rush add [-h] -p PACKAGE_NAME [-v VERSION_RANGE] [--exact] [--caret]
+                [--dev] [-c] [-n]
+                
+
+Blah.
```

Comment:
?

Reply:
Document that it's a tilde by default.

---
In reply to: [220731693](https://github.com/Microsoft/web-build-tools/pull/843#discussion_r220731693) [](ancestors = 220731693)